pkgs/applications/printing/braille-printer-app/default.nix

```
@@ -0,0 +1,43 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, autoreconfHook
+, pkg-config
+, cups
+, liblouis
+}:
+
+stdenv.mkDerivation rec {
+  pname = "braille-printer-app";
+  version = "2.0b1";
+
+  src = fetchFromGitHub {
+    owner = "OpenPrinting";
+    repo = "${pname}";
```

```suggestion
    repo = pname;
```
We can make this simpler.

---

resolved in rebased commits